src/main/java/gui/BaseScene.java

```diff
@@ -30,18 +34,28 @@ public BaseScene(final Controller crtl) {
      * @return topPane
      */
     protected Pane createTopPane() {
-        final int size = 100;
-        Text text = new Text("TrackScape");
-        text.setFont(Font.font("Edwardian Script ITC", size));
-        text.setFill(Color.BLACK);
-        text.setStroke(Color.LIGHTSLATEGREY);
-        text.setStrokeWidth(2);
+//        final int size = 100;
+//        Text text = new Text("TrackScape");
+//        text.setFont(Font.font("Edwardian Script ITC", size));
+//        text.setFill(Color.BLACK);
+//        text.setStroke(Color.LIGHTSLATEGREY);
+//        FlowPane textPane = new FlowPane();
+//        textPane.getChildren().add(text);
+//        textPane.setAlignment(Pos.CENTER);
```

Comment:
Why this commented code?

Reply:
Removed it. It was there to decide whether to change the text into the logo that is now there, but I kinda want to use this logo anyways.